README.md

```diff
@@ -99,6 +99,22 @@ DIGBot is capable of reacting when events are scheduled to remind people of thei
 Some channels the community would prefer to always have a free version of. This is the case for channels such as dedicated voice channels for certain games, for example if a member joins a voice channel for a game such as planetside DIGBot will react by creating a second voice channel for that game so there is still a free channel. Every-time a modular channel become populated or empty, DIGBot will respond by creating or deleting a channel as needed
 
 
+## Usage
```

Comment:
This I'm happy to alter on develop

Reply:
It definitely needs some improvement. I added it to at least have something on how to run it as the docker readme files was removed.